apps/experiments/models.py

```diff
@@ -1453,6 +1450,30 @@ def with_last_message_created_at(self):
         )
         return self.annotate(last_message_created_at=models.Subquery(last_message_created_at))
 
+    def with_first_message_created_at(self):
+        first_message_created_at = (
+            ChatMessage.objects.filter(
+                chat__experiment_session=models.OuterRef("pk"),
+            )
+            .order_by("created_at")
+            .values("created_at")[:1]
+        )
+        return self.annotate(first_message_created_at=models.Subquery(first_message_created_at))
+
+
+class ExperimentSessionObjectManager(models.Manager):
+    def for_chat_id(self, chat_id: str) -> list["ExperimentSession"]:
```

Comment:
Not really related to this PR, but it doesn't look like this method is used anymore, except by tests